src/us_reverse_geo/Client.js

```diff
@@ -0,0 +1,40 @@
+const Request = require("../Request");
+const SmartyResponse = require("./SmartyResponse");
+const buildInputData = require("../util/buildInputData");
+const keyTranslationFormat = require("../util/apiToSDKKeyMap").usReverseGeo;
+const {UndefinedLookupError} = require("../Errors.js");
+
+/**
+ * This client sends lookups to the SmartyStreets US Reverse Geo API, <br>
+ *     and attaches the results to the appropriate Lookup objects.
+ */
+class Client {
+	constructor(sender) {
+		this.sender = sender;
+	}
+
+	send(lookup) {
+		if (typeof lookup === "undefined") throw new UndefinedLookupError();
+
+		let request = new Request();
+		request.parameters = buildInputData(lookup, keyTranslationFormat);
+
+		return new Promise((resolve, reject) => {
+			this.sender.send(request)
+				.then(response => {
+					if (response.error) reject(response.error);
+
+					resolve(attachLookupResults(response, lookup));
+				})
+				.catch(reject);
+		});
+
+		function attachLookupResults(response, lookup) {
+			lookup.response = response.payload;
```

Comment:
If this is how we are setting the response, instead of using `SmartyResponse` then you can remove `SmartyResponse` completely (SmartyResponse.js and test_SmartyResponse.js)

Reply:
Same with the reference to `SmartyRepsonse` in test_Client.js